tests/gtest/aviflosslesstest.cc

```diff
@@ -64,26 +62,70 @@ void ReadImageSimple(const std::string& file_name, avifPixelFormat pixel_format,
   ASSERT_NE(file_format, AVIF_APP_FILE_FORMAT_UNKNOWN);
 }
 
+// Checks whether an image is grayscale. ASSERT_NO_FATAL_FAILURE should be used
+// when calling it.
+void IsGray(const std::string& path, bool& is_gray) {
```

Comment:
`IsGray` makes this sound like a function that returns a bool. You could say `GetIsGray` I guess. Otherwise you'd need to check `HasFailure()` separately.

Reply:
Done